lib/irb/lc/help-message

```diff
@@ -10,52 +10,53 @@
 #
 #
 Usage:  irb.rb [options] [programfile] [arguments]
-  -f                Suppress read of ~/.irbrc
-  -d                Set $DEBUG to true (same as `ruby -d')
-  -r load-module    Same as `ruby -r'
-  -I path           Specify $LOAD_PATH directory
-  -U                Same as `ruby -U`
-  -E enc            Same as `ruby -E`
-  -w                Same as `ruby -w`
-  -W[level=2]       Same as `ruby -W`
+  -f                Don't initialize from configuration file.
+  -d                Set $DEBUG and $VERBOSE to true (same as 'ruby -d').
+  -r load-module    Require load-module (same as 'ruby -r').
+  -I path           Specify $LOAD_PATH directory (same as 'ruby -I').
+  -U                Set external and internal encodings to UTF-8.
+  -E ex[:in]        Set default external and internal encodings
```

Comment:
`-E enc` became `-E ex[:in]`, which is terse. Perhaps we could repeat these short forms in parentheses after the written-out words in the help text?

```suggestion
  -E ex[:in]        Set default external (en) and internal (in) encodings
```

Reply:
I think there's a typo in this suggestion

```suggestion
  -E ex[:in]        Set default external (ex) and internal (in) encodings
```